pkg/cloudprovider/cloud.go

```diff
@@ -167,3 +167,7 @@ func (s ServiceAnnotation) GetValue(key string) (string, bool) {
 	val, ok := s[key]
 	return val, ok
 }
+
+const (
+	LBAnnotationAllowSourceRange = "service.beta.kubernetes.io/load-balancer-source-ranges"
```

Comment:
I meant "limbo" because the new net policy stuff applies policy to the underlying service, and this applies policy to the LB frontend, leaving NodePort as the only `type` that doesn't have some policy enforcement.  We *could* apply policy to node ports, it wouldn't even be that hard.  I don't think we need to, I just wanted to think about it.